### PR TITLE
Comparing --cache-min option

### PR DESCRIPTION
**Pull request by siddharthkp**
- [x] Clean installs
- [x] Multiple runs
- [x] Added --cache-min (`npm install --cache-min 9999999`)

My results with `express`
express

### CODE REVIEW

File: bin/npmvsyarn.js [hunk 1]
```diff
@@ -9,56 +9,73 @@ let prettyMs = require('pretty-ms'),
 
 let packageName = process.argv[2];
 
-// Check if yarn exists, if not install it.
```
Comment (supreetpal) [inline bin/npmvsyarn.js:12]: @siddharthkp you have accidentally removed this logic which checks if yarn already exists. If not, installs it.

File: bin/npmvsyarn.js [hunk 2]
```diff
@@ -9,56 +9,73 @@ let prettyMs = require('pretty-ms'),
...
     colWidths: [20, 10, 10],
-    style: {
-        head: ['yellow']
-    }
 });
 
-table.push(
-    ['npm', prettyMs(npmAddTime), prettyMs(npmRemoveTime)], ['yarn', prettyMs(yarnAddTime), prettyMs(yarnRemoveTime)]
-);
+// Install module using yarn add.
```
Comment (supreetpal) [inline bin/npmvsyarn.js:17]: This comment is not relevant over here. Should be moved to below rm node_modules.

File: bin/npmvsyarn.js [hunk 3]
```diff
@@ -9,56 +9,73 @@ let prettyMs = require('pretty-ms'),
...
+
+sh.exec('npm cache clean', {silent: true});
+sh.exec('rm node_modules', {silent: true});
```
Comment (supreetpal) [inline bin/npmvsyarn.js:20]: rm -rf should be a better option.

File: bin/npmvsyarn.js [hunk 4]
```diff
@@ -9,56 +9,73 @@ let prettyMs = require('pretty-ms'),
...
+	sh.exec('npm uninstall ' + packageName, { silent: true });
+	let npmRemoveTime = Date.now() - timeStamp;
+	console.log('🚀 Time taken by npm to uninstall ' + clc.red(prettyMs(npmRemoveTime)) + '');
+
+	table.push(['npm run ' + i, prettyMs(npmAddTime), prettyMs(npmRemoveTime)]);
+}
+
+// Install module using npm install with cache-min
+sh.exec('npm cache clean', {silent: true});
+sh.exec('rm node_modules', {silent: true});
```
Comment (supreetpal) [inline bin/npmvsyarn.js:62]: rm -rf

**Review by supreetpal — CHANGES_REQUESTED** [review]
Mostly looks great! I have left a few comments, which need to be resolved before I can merge. 
Please also move this PR to https://github.com/supreetpal/npmvsyarn/tree/feature/multiple-runs instead of master.

**Comment by supreetpal** [comment]
rm -rf node_modules is also not a safe command to run. It will remove the existing node_modules folder of the user where this command is run. We need to save the existing one in temp then move it back later. Instead of simple clearing it. I will add this logic, after we merge this so you can just make the changes which have been requested in the review.

**Comment by siddharthkp** [comment]
@supreetpal I didn't expect that you would want to merge this 😄
Was just showing the code for comparison

Although, if you're interested I can help you out with the other features

**Pull request closed without merging**